packages/roosterjs-content-model-core/lib/editor/Editor.ts

```diff
@@ -104,23 +105,25 @@ export class Editor implements IEditor {
 
         switch (mode) {
             case 'connected':
-                return core.api.createContentModel(core, {
-                    processorOverride: {
-                        table: tableProcessor, // Use the original table processor to create Content Model with real table content but not just an entity
-                    },
-                });
+                return core.api.createContentModel(core);
 
             case 'disconnected':
-                return cloneModel(core.api.createContentModel(core), {
-                    includeCachedElement: this.cloneOptionCallback,
-                });
+                return cloneModel(
+                    core.api.createContentModel(core, {
+                        processorOverride: {
+                            table: tableProcessor,
```

Comment:
why are we adding this tableProcessor?

Reply:
I moved this from the "connected" part to "disconnected"

For fidelity and performance reason, today we are using an overridden table processor by default in editor to treat table as a fake entity if there is no selection in it. But here when we want to create a disconnected model, we should set it back to original table processor to really create model for table, but not an entity.

I moved it because with this change, if there is any override option passed in, we will not reuse existing cached model. So for better performance, I removed the override from "connected" part.